lib/resources/styles.css

```diff
@@ -10,15 +10,84 @@
 .secondary-text-color  { color: #727272; }
 .divider-color         { border-color: #B6B6B6; }
 
-html {
-  position: relative;
-  min-height: 100%;
+/* for layout */
+html,
+body {
+  margin: 0;
+  padding: 0;
+  height: 100%;
+}
+body {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+header {
+  flex: 0 0 56px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-left: 30px;
+}
+header ol {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+header ol li {
+  display: inline;
+}
+header form {
+  display: flex;
+  flex: 1;
+  justify-content: flex-end;
+  padding-right: 30px;
+  
+}
+footer {
+  flex: 0 0 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 8px;
```

Comment:
hmm, the different - much smaller font - is distracting, and there's not enough separation between the content and its border.